### PR TITLE
add optional team id to create rule group request

### PR DESCRIPTION
**Pull request by daniellelara**
allows rule to be created  for a team (specifically in creating a new team) when signed into another team, when implemented, will check if team id exists as a param, and if not, will take as it does now from auth context

### CODE REVIEW

**Review by deniszo — APPROVED** [review]

File: com/coralogix/rules/v1/rule_groups_service.proto [hunk 1]
```diff
@@ -98,6 +98,7 @@ message CreateRuleGroupRequest {
   repeated CreateRuleSubgroup rule_subgroups = 7;
 
   google.protobuf.UInt32Value order = 8;
+  optional TeamId team_id = 9;
```
Comment (galrose) [inline com/coralogix/rules/v1/rule_groups_service.proto:101]: why not?
```suggestion
  google.protobuf.UInt32Value team_id = 9;
```

**Review by galrose — CHANGES_REQUESTED** [review]

File: com/coralogix/rules/v1/rule_groups_service.proto (hunk 1 above)
Comment (galrose) [reply com/coralogix/rules/v1/rule_groups_service.proto:101]: and you dont need to optional proto3 is optional by default :)

**Review by galrose — DISMISSED** [review]

File: com/coralogix/rules/v1/rule_groups_service.proto (hunk 1 above)
Comment (daniellelara) [reply com/coralogix/rules/v1/rule_groups_service.proto:101]: in other protobuf definitions we have used teamId as a new message type

Comment (galrose) [reply com/coralogix/rules/v1/rule_groups_service.proto]: oh I see, well its a personal preference not that important

**Review by galrose — APPROVED** [review]

**Pull request merged**